include/swift/SIL/SILInstruction.h

```diff
@@ -52,6 +53,9 @@ class ValueDecl;
 class VarDecl;
 class FunctionRefInst;
 
+SILValue getUndefValue(SILType Ty);
```

Comment:
Yes, all those Undef related function are not needed anymore. I forgot to remove them.